tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -144,13 +144,16 @@ def __init__(self, marionette):
         update = Wait(self.marionette).until(expected.element_present(*self._update_locator))
         Wait(self.marionette).until(lambda m: update.location['y'] == 0 and update.is_displayed())
 
-    def tap_update(self, return_details=True):
+    def tap_update(self, return_details=True, return_class=None):
```

Comment:
[Like here](https://github.com/mozilla-b2g/gaia/pull/33392#discussion_r46123478), what happens if you call this function with `return_details=True` and `return_class='Merge'`? 

The intent of the function doesn't look clear. In order to remove this ambiguity, let's just use 1 argument: `return_class`.